### PR TITLE
Update article.md

### PR DESCRIPTION
**Pull request by paroche**
As per issue comment thread #1282: 
Clarifying "current code" for zero delay setInterval. 
Plus minor edit of ordering of terms for greater consistency.
Plus minor grammatical fixes, etc.

### CODE REVIEW

**Comment by iliakan** [comment]
Thanks, that's much better. Although, there's a hesitation.

If we replace "current code" with "current script", does it really make things clear?

There's an additional chapter http://javascript.info/event-loop that should help, but it comes much later.

**Comment by paroche** [comment]
I think it totally does. Code is very general. Script is what lies between <script> and </script>. 
At least I personally didn't know what "current code" meant,. Maybe others are similarly slow. :->


-----Original Message-----

File: 1-js/06-advanced-functions/08-settimeout-setinterval/article.md [hunk 1]
```diff
@@ -286,10 +286,10 @@ For server-side JavaScript, that limitation does not exist, and there exist othe
...
-- To cancel the execution, we should call `clearInterval/clearTimeout` with the value returned by `setInterval/setTimeout`.
-- Nested `setTimeout` calls is a more flexible alternative to `setInterval`, allowing to set the time *between* executions more precisely.
-- Zero delay scheduling with `setTimeout(func, 0)` (the same as `setTimeout(func)`) is used to schedule the call "as soon as possible, but after the current code is complete".
+- Methods `setTimeout(func, delay, ...args)` and `setInterval(func, delay, ...args)` allow us to run the `func` once/regularly after `delay` milliseconds.
+- To cancel the execution, we should call `clearTimeout/clearInterval` with the value returned by `setTimeout/setInterval`.
+- Nested `setTimeout` calls is a more flexible alternative to `setInterval`, allowing us to set the time *between* executions more precisely.
+- Zero delay scheduling with `setTimeout(func, 0)` (the same as `setTimeout(func)`) is used to schedule the call "as soon as possible, but after the current code (script) is complete".
```
Comment (iliakan) [inline 1-js/06-advanced-functions/08-settimeout-setinterval/article.md]: - Zero delay scheduling with `setTimeout(func, 0)` (the same as `setTimeout(func)`) is used to schedule the call "as soon as possible, but after the current script is complete".

File: 1-js/06-advanced-functions/08-settimeout-setinterval/article.md [hunk 2]
```diff
@@ -239,7 +239,7 @@ There's a side-effect. A function references the outer lexical environment, so,
 
 There's a special use case: `setTimeout(func, 0)`, or just `setTimeout(func)`.
 
-This schedules the execution of `func` as soon as possible. But scheduler will invoke it only after the current code is complete.
+This schedules the execution of `func` as soon as possible. But the scheduler will invoke it only after the current code (i.e. the currently executing script) is complete.
```
[the remark this answers is not in the document]
Comment (paroche) [reply 1-js/06-advanced-functions/08-settimeout-setinterval/article.md]: I think that would be fine, better even, but I seem to recall that there are other places in the article where you refer to the "current code", which might then also be better if changed to say "current script".

**Pull request merged**